indra/sources/bel/processor.py

```diff
@@ -334,12 +334,12 @@ def _get_evidence(self, u_data, v_data, k, edge_data):
         ev_text = edge_data.get(pc.EVIDENCE)
         ev_citation = edge_data.get(pc.CITATION)
         ev_pmid = None
+        ev_ref = None
```

Comment:
this change was necessary to deal with BELGraph edges that are programmatically generated, that could potentially be missing a citation

Reply:
There was actually a comment about this below that I deleted in 08879ef